test/types/authn.module.test-d.ts

```diff
@@ -22,47 +22,37 @@ import {
   OktaAuthHttpInterface,
   FingerprintAPI
 } from '@okta/okta-auth-js/authn';
-
-import { expectType, expectAssignable, expectError } from 'tsd';
+import { expect } from 'tstyche';
 
 const authClient = new OktaAuth({issuer: 'https://{yourOktaDomain}/oauth2/default'});
-expectAssignable<OktaAuthTxInterface>(authClient);
-expectAssignable<OktaAuthHttpInterface>(authClient);
-expectAssignable<OktaAuthCoreInterface>(authClient);
-
-expectType<AuthnTransactionAPI>(authClient.tx);
-expectType<AuthnTransactionAPI>(authClient.authn);
-expectType<FingerprintAPI>(authClient.fingerprint);
+expect<OktaAuthTxInterface>().type.toBeAssignable(authClient);
+expect<OktaAuthHttpInterface>().type.toBeAssignable(authClient);
+expect<OktaAuthCoreInterface>().type.toBeAssignable(authClient);
 
-// does not include OAuth
```

Comment:
authn export does include Core, therefore these tests _should_ be failing as written. Removed in PR

Reply:
@jaredperreault-okta 
Maybe correct and include tests instead of removing?
```js
// does include OAuth
expect(await authClient.token.parseFromUrl()).type.toEqual<TokenResponse>();
//or
expect(authClient).type.toHaveProperty('token');

// does include Core
expect(authClient.authStateManager.getAuthState()).type.toEqual<AuthState | null>();
//or
expect(authClient).type.toHaveProperty('authStateManager');
```